.circleci/create_circleci_config.py

```diff
@@ -372,16 +372,16 @@ def create_circleci_config(folder=None):
             test_list = f.read()
     else:
         test_list = []
-    if len(test_list) > 0:
-        jobs.extend(REGULAR_TESTS)
+    # if len(test_list) > 0:
```

Comment:
Don't review this file. It's just for running (only) pipeline tests - I will remove the change before merge.